### ISSUE TITLE
my flutter is giving Error when i run the project

### ISSUE THREAD

**Issue by prince-winter** [flutter/flutter#60326]
12:33:32 AM: Executing task...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.android.tools.build:gradle:5.6.2.
     Required by:
         project :
      > Could not resolve com.android.tools.build:gradle:5.6.2.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/5.6.2/gradle-5.6.2.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/5.6.2/gradle-5.6.2.pom'.
               > failed: Connection refused: connect
      > Could not resolve com.android.tools.build:gradle:5.6.2.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/5.6.2/gradle-5.6.2.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/5.6.2/gradle-5.6.2.pom'.
               >  failed: Connection refused: connect
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72.
     Required by:
         project :
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.72/kotlin-gradle-plugin-1.3.72.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.72/kotlin-gradle-plugin-1.3.72.pom'.
               >  failed: Connection refused: connect
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72.
         > Could not get resource 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.72/kotlin-gradle-plugin-1.3.72.pom'.
            > Could not GET 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.72/kotlin-gradle-plugin-1.3.72.pom'.
               >  failed: Connection refused: connect
   > Could not resolve com.google.gms:google-services:4.3.3.
     Required by:
         project :
      > Could not resolve com.google.gms:google-services:4.3.3.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.3/google-services-4.3.3.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.3/google-services-4.3.3.pom'.
               failed: Connection refused: connect
      > Could not resolve com.google.gms:google-services:4.3.3.
         > Could not get resource 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.3/google-services-4.3.3.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.3/google-services-4.3.3.pom'.
           ] failed: Connection refused: connect

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 30s
12:34:04 AM: Task execution finished.

**Comment by HemilTheRebel** [comment]
Do you have an active internet connection? I am no expert but it seems like gradle is unable to download dependencies

**Comment by VladyslavBondarenko** [comment]
@prince-winter 
be sure you have internet connection when building, if being behind a firewall your case - using mirror sites https://flutter.dev/community/china#configuring-flutter-to-use-a-mirror-site should help. 
Have you solved it?

**Comment by jobic10** [comment]
If you created the project offline and was trying to run the app the very first time, consider connecting to a network to download dependencies.
Otherwise, still get connected to a network.

**Comment by prince-winter** [comment]
yeah i have fixed it. what i just did was to reset my android studio and remove my others downloads and download another one. BOOM!!! it compiled.thanks for the help
